release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -805,6 +805,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+A default storage limit of 30% has been added for data collected by the `oc adm must-gather` command. To adjust the default storage limit value a new flag has been added to the OpenShift CLI.
```

Comment:
* Probably should clarify what it is 30% of
* Better to use active voice, and write it in the perspective of the reader taking action. Added a suggestion but you can play with it however you'd like

```suggestion
A default limit of 30% of the storage capacity of the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```

Reply:
Thank you!